system/greenhouse-ccloud/templates/thanos-pluginpreset.yaml

```
@@ -16,6 +16,11 @@ spec:
           paths:
             - path: /
               pathType: ImplementationSpecific
+    - name: thanos.query.ingress.grpc.tls
+      value:
+        - hosts:
+            - "thanos-grpc.{{ $ingressDomain }}"
+          secretName: {{ printf "tls-%s" (print "thanos-grpc." $ingressDomain | replace "." "-") }}
```

Some suggestion to make it cleaner. 
```suggestion
          secretName: {{ printf "tls-thanos-grpc-%s" $ingressDomain | replace "." "-" }}
```